### DISCUSSION TITLE
Help, when in terminal I use cat ~/.chia/mainnet/log/debug.log

### DISCUSSION THREAD

**Discussion by cryptofraga** [Chia-Network/chia-blockchain#6368] [Support]
hi, when i use this code cat + ~ / .chia / mainnet / log / debug.log to check challenge times. I get a sequence of errors. I do not understand. How can I solve this? Thanks.

https://ibb.co/hcnh5r3

**Comment by BrandtH22** [comment]
@luisfgbb , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - more information about how to check things are working can be found here: https://docs.chia.net/check-if-things-are-working/

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
